features/user_authentication.feature

```diff
@@ -3,12 +3,11 @@ Feature: User is authenticated for both sign up and login.
   In order to access the site
   We need to be able to sign up or log in
 
-
   Background:
     Given the following user exists
-      |  email    | password    | password_confirmation |
+      | email    | password    | password_confirmation |
       | jf@ca.com | password123 | password123           |
-    And I am on the "Landing" page
+    And I am on the Landing page
```

Comment:
So if you implement the above suggestion you can keep this step the way it was before